quadrat/inc/block-patterns.php

```diff
@@ -27,6 +27,47 @@ function quadrat_register_block_patterns() {
 				)
 			);
 
+			register_block_pattern(
+				'quadrat-blocks/example',
+				array(
+					'title'      => __( 'Latest Episodes', 'quadrat' ),
+					'categories' => array( 'quadrat' ),
+					'content'    => '<!-- wp:columns {"align":"wide"} -->
+					<div class="wp-block-columns alignwide"><!-- wp:column -->
+					<div class="wp-block-column"><!-- wp:image {"id":1438,"sizeSlug":"large","linkDestination":"none","className":"image-no-margin mb-0"} -->
+					<figure class="wp-block-image size-large image-no-margin mb-0"><img src="http://freethemes.local/wp-content/uploads/2021/04/Screen-usage-1024x623.jpg" alt="' . esc_html__( 'Screen usage', 'quadrat') . '" class="wp-image-1438"/></figure>
```

Comment:
We probably need a better alt text than this

Reply:
Also, @beafialho should we keep the lorem ipsum or do we want other kind of text for these?

Reply:
Yes, @MaggieCabrera here's the alt text for the images of the patterns. I'm also writing alt text for all the other abstract illustrations [here](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=111%3A34).

"Illustration of two women."
![Illustration of two women](https://user-images.githubusercontent.com/65220155/115046915-e7d75580-9ecf-11eb-8cb1-cc4aa7c750b3.jpg)

"Illustration of an introspective woman."
![Illustration of an introspective woman](https://user-images.githubusercontent.com/65220155/115046920-e9088280-9ecf-11eb-8be0-a426f8a7a58a.jpg)

"Illustration of a woman working on a laptop."
![Illustration of a woman working on a laptop](https://user-images.githubusercontent.com/65220155/115046923-ea39af80-9ecf-11eb-898e-b9c15d12c45f.jpg)

"Illustration of a woman climbing steps."
![Illustration of a woman climbing steps](https://user-images.githubusercontent.com/65220155/115046926-ead24600-9ecf-11eb-8095-757c16160dbe.jpg)

> Also, @beafialho should we keep the lorem ipsum or do we want other kind of text for these?

I've replaced the lorem ipsum [here](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=443%3A5).

Reply:
Thanks so much!